src/flamegraph.cpp

```diff
@@ -680,6 +799,23 @@ void FlameGraph::saveSvg(const QString& fileName) const
     m_rootItem->setBrush(oldBrush);
 }
 
+void FlameGraph::setColorScheme(ColorScheme _scheme)
+{
+    scheme = _scheme;
+
+    std::function<void(FrameGraphicsItem*)> traverseFunction = [&traverseFunction](FrameGraphicsItem* item) {
```

Comment:
make this a real recursive function instead of forcing the memory allocation via `std::function` here